ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -1379,6 +1379,18 @@ router bgp {{ router_bgp.as }}
 {%         endif %}
 {%         if vrf.evpn_multicast is arista.avd.defined(true) %}
       evpn multicast
+{%             if vrf.evpn_multicast_gateway_dr_election_algorithm.hrw is arista.avd.defined(true) or vrf.evpn_multicast_gateway_dr_election_algorithm.modulus is arista.avd.defined(true)
```

Comment:
vrf.evpn_multicast_gateway_dr_election_algorithm store this long data into an variable

Reply:
Done.